cylc/flow/simulation.py

```diff
@@ -46,23 +100,17 @@ def configure_sim_modes(taskdefs, sim_mode):
     for tdef in taskdefs:
         # Compute simulated run time by scaling the execution limit.
         rtc = tdef.rtconfig
-        sleep_sec = get_simulated_run_len(rtc)
-
-        rtc['execution time limit'] = (
-            sleep_sec + DurationParser().parse(str(
-                rtc['simulation']['time limit buffer'])).get_seconds()
-        )
 
-        rtc['simulation']['simulated run length'] = sleep_sec
         rtc['submission retry delays'] = [1]
 
-        # Generate dummy scripting.
-        rtc['init-script'] = ""
-        rtc['env-script'] = ""
-        rtc['pre-script'] = ""
-        rtc['post-script'] = ""
-        rtc['script'] = build_dummy_script(
-            rtc, sleep_sec) if dummy_mode else ""
+        if dummy_mode:
```

Comment:
Note, remember not to permit `[runtime][<namespace>]run mode = dummy`.

Reply:
This method is no longer being called at runtime in this version - some of the methods it uses are called to recalculate values in the `mode_settings` object.